api/tools/src/main/java/org/pmiops/workbench/tools/PopulateOpsUserAffiliations.java

```diff
@@ -0,0 +1,209 @@
+package org.pmiops.workbench.tools;
+
+import com.opencsv.CSVReader;
+import java.io.FileReader;
+import java.io.IOException;
+import java.util.List;
+import java.util.logging.Logger;
+import java.util.stream.Collectors;
+import java.util.stream.StreamSupport;
+import org.apache.commons.cli.CommandLine;
+import org.apache.commons.cli.DefaultParser;
+import org.apache.commons.cli.Option;
+import org.apache.commons.cli.Options;
+import org.pmiops.workbench.db.dao.InstitutionDao;
+import org.pmiops.workbench.db.dao.UserDao;
+import org.pmiops.workbench.db.dao.VerifiedInstitutionalAffiliationDao;
+import org.pmiops.workbench.db.model.DbInstitution;
+import org.pmiops.workbench.db.model.DbUser;
+import org.pmiops.workbench.db.model.DbVerifiedInstitutionalAffiliation;
+import org.pmiops.workbench.model.InstitutionalRole;
+import org.springframework.boot.CommandLineRunner;
+import org.springframework.context.annotation.Bean;
+
+/**
+ * Populate the verified_institutional_affiliation table for Ops users from a CSV input file.
+ *
+ * <p>NOTE: input file must be located in the current directory or a subdirectory.
+ *
+ * <p>Example execution:
+ *
+ * <pre>
+ * ./project.rb populate-ops-user-affiliations \
```

Comment:
Our ops stack requires us to use Ruby to run Java tools, and errors are often missing or incorrect, so I spend far too much time hunting down correct command lines.  This will be a major help, if only to myself!